.github/workflows/preview-pull-request.yml

```diff
@@ -31,10 +31,8 @@ jobs:
         # Note, do not add python scripts here due to easy private key access.
         run: |
           rm spec/index.bs wgsl/index.bs explainer/index.bs
-          wget -O spec/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/spec/index.bs
-          wget -O wgsl/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/wgsl/index.bs
-          wget -O explainer/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/explainer/index.bs
-          wget -O correspondence/index.bs https://raw.githubusercontent.com/${{ github.event.workflow_run.head_repository.full_name }}/${{ github.event.workflow_run.head_sha }}/correspondence/index.bs
+          git fetch https://github.com/${{ github.event.workflow_run.head_repository.full_name }} ${{ github.event.workflow_run.head_sha }}
+          git checkout ${{ github.event.workflow_run.head_sha }} {spec,wgsl,explainer,correspondence}/{index.bs,img/}
```

Comment:
Can we do *.mmd.svg if that is possible? (btw sorry for dispatching https://github.com/gpuweb/gpuweb/issues/3386#issuecomment-1230701439 comment early).

Reply:
Just checked https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefpathspecapathspec pathspec, should be possible

Reply:
There can be other diagram filetypes (not just generated ones) too so I filtered it to bs,svg,png,jpg.